packages/artifact/LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 GitHub
```

Comment:
Shouldn't there be a line with `The MIT License (MIT)` at the top of each file?

Our licenses for some other repos have it included: https://github.com/actions/download-artifact/blob/c3f5d00c8784369c43779f3d2611769594a61f7a/LICENSE#L2

Reply:
This should be the license text, it is taken from the root license file in this repo.

https://opensource.org/licenses/MIT

There's no harm in adding that extra line, but I don't see a reason to.